guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -8,16 +8,7 @@ Perform the following procedure if you want to disable {Cockpit} on {Project}.
 +
```

Comment:
Good point. Updated, but I wonder: does it even need a bullet?

Reply:
The style guide says it does need a bullet so I'd recommend to go with that. Also, it would be consistent with the [module on enabling cockpit](https://github.com/theforeman/foreman-documentation/blob/f11d45fcd533e60dd3bbfb963dcefff2df699d9f/guides/common/modules/proc_enabling-cockpit-on-server.adoc?plain=1#L8).